sdcm/cluster.py

```diff
@@ -2831,8 +2831,8 @@ def terminate_node(self, node):
         node.destroy()
 
     def get_db_auth(self):
-        user = self.params.get('authenticator_user', default=None)
-        password = self.params.get('authenticator_password', default=None)
+        user = self.params.get('authenticator_user')
```

Comment:
set the default in `test_default.yaml`

Reply:
I don't thins it's correct. We don't want to use credentials in every test

Reply:
no set to "" (empty string)